sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -94,6 +101,43 @@ FormRecognizerClient formRecognizerClient = new FormRecognizerClientBuilder()
 credential.update("{new_key}");
 ```
 
+#### Create a Form Recognizer client with Azure Active Directory credential
+Azure SDK for Java supports an Azure Identity package, making it easy to get credentials from Microsoft identity
+platform. 
+
+Authentication with AAD requires some initial setup:
+* Add the Azure Identity package
+
+[//]: # ({x-version-update-start;com.azure:azure-identity;dependency})
+```xml
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-identity</artifactId>
+    <version>1.0.6</version>
+</dependency>
+```
+[//]: # ({x-version-update-end})
+* [Register a new Azure Active Directory application][register_AAD_application]
+* [Grant access][grant_access] to Form Recognizer by assigning the `"Cognitive Services User"` role to your service principal.
+
+After setup, you can choose which type of [credential][azure_identity_credential_type] from azure.identity to use. 
+As an example, [DefaultAzureCredential][wiki_identity] can be used to authenticate the client:
+Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables: 
+AZURE_CLIENT_ID, AZURE_TENANT_ID, AZURE_CLIENT_SECRET. 
+
+Authorization is easiest using [DefaultAzureCredential][wiki_identity]. It finds the best credential to use in its
+running environment. For more information about using Azure Active Directory authorization with Form Recognizer, please
+refer to [the associated documentation][aad_authorization].
+
+<!-- embedme ./src/samples/java/com/azure/ai/formrecognizer/ReadmeSamples.java#L51-L55 -->
+```java
+TokenCredential credential = new DefaultAzureCredentialBuilder().build();
+FormRecognizerClient formRecognizerClient = new FormRecognizerClientBuilder()
+    .endpoint("{endpoint}")
+    .credential(credential)
+    .buildClient();
+```
+
```

Comment:
nit: I can't comment on the line, but I notice the "More sample code" [section](https://github.com/Azure/azure-sdk-for-java/blob/aacc161d72d7a7cfe9935fb759e1d0f9e24b41bf/sdk/formrecognizer/azure-ai-formrecognizer/README.md#more-sample-code) is displaying the samples a little weirdly, i.e: 

Recognize receipts: RecognizeReceipts, (async)

Can you get rid of the comma?

Reply:
updated